### DISCUSSION TITLE
waline支持Gitee的数据存储库吗？

### DISCUSSION THREAD

**Discussion by AkarinLiu** [walinejs/waline#286] [Ideas]
既然Github作为存储库访问不稳定，那为何不支持Gitee的数据存储库呢？

**Comment by lizheming** [comment]
技术上可以实现，欢迎感兴趣的提个 PR？整体其实参考 Github 的改一下就行

**Comment by Mister-Hope** [comment]
我过段时间有空看一下这个问题，不过这种通过仓库存储来代替数据库的方案本身性能就相对标准的数据库要差。

**Comment by AnzhiZhang** [comment]
既然Github作为存储库访问不稳定，为什么不考虑服务器呢
